content/english/ml-machine-learning/01-regression.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "What is regression?"
+description: "Make computers learn to predict outcomes."
+prereq: "Python"
+icon: ""
+draft: false
+weight: 1
+---
+
+# What is regression?
+Regression is a technique to model the relationship between a feature (independent variables) and a variable feature (dependent variables). It helps us understand how the value of a dependent variable is changing based on the value of the independent variable. When applied properly it helps us predict values 😯!
+
+|![Regression](./resources/regression.png)|
+|:--:|
+|Regression|
+
+Regression gives a line or a curve by plotting dependent and independent variables. When we apply regression, we need to come up with a mathematical model that can explain the behavior we are seeing. If the mathematical model we use doesn't represent the data accordingly we need to find another model to use.
+
+## Example 1: Predicting the Weather Temperature
+For example, say you need to predict the temperature of a place. What do you need?
+
+- Altitude
+- Location
+- Month
+- Year
+- Humidity
+- Atmospheric Pressure
+
+The **temperature** is the dependent variable while the other properties we have mentioned above are the independent variables.
+
+## Fitting Your Model
+To fit a model is to make sure that the model you are using represents the information you have.
+
+### Excercise 1
+Suppose that someone wants to know the type of shapes they have based on the area of the shape. 
+
+1. Is that even possible? Can someone know what shape they have based on the area?
+{{% expand "**Click to show answer**" %}}  
+***No, as you can quickly guess no relation exists between the **area** and the type of shape.***
```

Comment:
Give an example: "for example, a square of side length 3 has an area of 9 (3 x 3 = 9), but so does a triangle with width of 6 and height of 3 ((6 x 3)/ 2 = 9)"